dbt/include/teradata/macros/utils/generate_series.sql

```diff
@@ -0,0 +1,30 @@
+{# Overriding because the original implementation used 'WITH'. Since this query is used in 'date_spine' macro #}
```

Comment:
Don't we require add documentation about generate_series.sql in ReadMe.md file

Reply:
Is this supports only numbers for generating series or dates aswell?

Reply:
Yes we need to add them to Readme, will add.
Thanks for reminding.
And this macros is not for dates. It is to generate series of numbers from 1 to the chosen limit.